src/data-types/mailstream_cfstream.c

```diff
@@ -972,6 +972,15 @@ int mailstream_cfstream_set_ssl_enabled(mailstream * s, int ssl_enabled)
       case MAILSTREAM_CFSTREAM_SSL_LEVEL_TLSv1:
         CFDictionarySetValue(settings, kCFStreamSSLLevel, kCFStreamSocketSecurityLevelTLSv1);
         break;
+      case MAILSTREAM_CFSTREAM_SSL_LEVEL_TLSv1_1:
```

Comment:
Those constants are not defined anywhere.
You need to add them.

Reply:
Ah, I see that's https://github.com/dinhviethoa/libetpan/pull/374

Reply:
Fix order to reflect enum type declaration.